utils/index.js

```diff
@@ -25,53 +26,69 @@ const getHexProof = (list, address) => {
 
 /**
  * Check whether wallet address is part of the merkle tree
- * @param {string[]} list 
- * @param {string} address 
+ * @param {string[]} list
+ * @param {string} address
  * @returns {bool}
  */
 const checkWhitelisted = (list, address) => {
 	const merkleTree = getMerkeTree(list)
 	const hexProof = merkleTree.getHexProof(ethers.utils.keccak256(address))
-	return merkleTree.verify(hexProof, ethers.utils.keccak256(address), merkleTree.getRoot())
+	return merkleTree.verify(
+		hexProof,
+		ethers.utils.keccak256(address),
+		merkleTree.getRoot()
+	)
 }
 
-const copyToClipboard = async function(value) {
-    await navigator.clipboard.writeText(value)
-    this.$bvToast.toast('Copied to clipboard!', {
-        title: 'Clipboard',
-        variant: 'info',
-    })
+const copyToClipboard = async function (value) {
+	await navigator.clipboard.writeText(value)
+	this.$bvToast.toast('Copied to clipboard!', {
+		title: 'Clipboard',
+		variant: 'info',
+	})
 }
 
 /**
- * 
- * @param {(int|string)} chainId 
- * @param {('nonstatic'|'batch')} [type] 
- * @returns 
+ *
+ * @param {(int|string)} chainId
+ * @param {('nonstatic'|'batch')} [type]
+ * @returns
  */
 const getProvider = (chainId, type = null) => {
-    const providerUrl = CHAINID_CONFIG_MAP[chainId.toString()].rpcUrls[0]
-    switch(type) {
-        case 'nonstatic': return new ethers.providers.JsonRpcProvider(providerUrl)
-        case 'batch': return new ethers.providers.JsonRpcBatchProvider(providerUrl)
-        default: return new ethers.providers.StaticJsonRpcProvider(providerUrl)
-    }
+	const providerUrl = CHAINID_CONFIG_MAP[chainId.toString()].rpcUrls[0]
+	switch (type) {
+		case 'nonstatic':
+			return new ethers.providers.JsonRpcProvider(providerUrl)
+		case 'batch':
+			return new ethers.providers.JsonRpcBatchProvider(providerUrl)
+		default:
+			return new ethers.providers.StaticJsonRpcProvider(providerUrl)
+	}
 }
 
 /**
- * 
- * @param {int} delay - delay in milliseconds 
- * @returns 
+ *
+ * @param {int} delay - delay in milliseconds
+ * @returns
  */
 const wait = (delay) => {
 	return new Promise((resolve) => setTimeout(resolve, delay))
 }
 
+/**
+ *
+ * @param {int} chainId
+ * @returns {bool}
+ */
+const isChainSupportedByOS = (chainId) => {
+	return Object.values(OS_SUPPORTED_CHAINS).includes(chainId)
```

Comment:
then here just list the chains that OS currently supports e.g. 
[ CHAIN_IDS.Ethereum, ...].includes(chainId)